types/koa-bouncer/index.d.ts

```diff
@@ -0,0 +1,93 @@
+// Type definitions for koa-bouncer 6.0.4
+// Project: https://github.com/danneu/koa-bouncer
+// Definitions by: maruware <https://github.com/maruware>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { Middleware, Context } from 'koa';
+
+declare namespace KoaBouncer {
+    export class ValidatorError extends Error {
+        name: string;
+        message: string;
+        bouncer: {
+            key: string,
+            message: string
+        }
+        constructor(key: string, message: string);
+    }
+
+    export class Validator {
+        constructor(props: {ctx: Context, key: string, vals: any})
+        required(tip?: string): Validator
+        optional(): Validator
+        check(result: any, tip?: string): Validator
+        checkNot(result: any, tip?: string): Validator
+        checkNot(result: any, tip?: string): Validator
```

Comment:
This is duplicated. Also `isNotIn` is duplicated.

Reply:
Thanks for reviewing!
I fixed it.